src/index.js

```diff
@@ -4,14 +4,19 @@ import './index.css'
 import App from './App'
 import { BrowserRouter, Routes, Route } from 'react-router-dom'
 import { AuthProvider } from './context/AuthContext'
+import { ChakraProvider } from '@chakra-ui/react'
 
 const root = ReactDOM.createRoot(document.getElementById('root'))
 root.render(
 	<React.StrictMode>
 		<AuthProvider>
 			<BrowserRouter>
 				<Routes>
-					<Route path="/*" element={<App />} />
+					<Route path="/*" element={
+						<ChakraProvider>
```

Comment:
Since you wrapped the whole app in <ChakraProvider> (which is better), please remove the <ChakraProvider> in the src/components/Login.js component. It's no longer needed in there and it shouldn't be on 2 places.

Reply:
Done